src/components/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { ChangeEvent, FunctionComponent, useEffect } from 'react';
+
+import autosizer from 'autosize';
+import classNames from 'classnames';
+
+import { useCallbackRef } from '../../hooks/useCallbackRef';
+
+export interface TextAreaProps {
+	id?: string;
+	rows?: number;
+	disabled?: boolean;
+	placeholder?: string;
+	value?: string;
+	size?: 'small' | 'medium' | 'large';
+	autosize?: boolean;
+	onChange?: (value: string) => void;
+}
+
+export const TextArea: FunctionComponent<TextAreaProps> = ({
+	id,
+	rows,
+	disabled = false,
+	placeholder,
+	value = '',
+	size,
+	autosize = false,
+	onChange = () => {},
+}: TextAreaProps) => {
```

Comment:
`: TextAreaProps` can be removed as it is already set in the `FunctionComponent`